releasenotes/notes/bosonic-operator-and-mapper-45bfde873f092681.yaml

```diff
@@ -0,0 +1,24 @@
+---
+features:
+  - |
+    Introduced a new feature that implements the bosonic operators ``BosonicOp``.
```

Comment:
sphinx uses the following syntax to link class names to the corresponding API ref:

```suggestion
    Introduced a new feature that implements the bosonic operators :class:`.BosonicOp`.
```

Reply:
Solved